src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Android/AndroidGetDeviceCommandArguments.cs

```diff
@@ -25,15 +27,18 @@ public string AppPackagePath
         /// If specified, attempt to run on a compatible attached device, failing if unavailable.
         /// If not specified, we will open the apk using Zip APIs and guess what's usable based off folders found in under /lib
         /// </summary>
-        public string? DeviceArchitecture { get; set; }
+        public IEnumerable<string> DeviceArchitecture => _deviceArchitecture;
 
         protected override OptionSet GetCommandOptions() => new()
         {
             { "app|a=", "Path to already-packaged app",
                 v => AppPackagePath = RootPath(v)
             },
-            { "device-arch=", "If specified, forces running on a device with given architecture (x86, x86_64, arm64-v8a or armeabi-v7a). Otherwise inferred from supplied APK",
-                v => DeviceArchitecture = v
+            {
+                "device-arch=",
+                "If specified, forces running on a device with given architecture (x86, x86_64, arm64-v8a or armeabi-v7a). Otherwise inferred from supplied APK. " +
+                "Can be used more than once.",
+                v => _deviceArchitecture.Add(v)
```

Comment:
If we can only have these values, should we validate what we received to be one of them?

Reply:
validation added